### PR TITLE
apply styling

### PR DESCRIPTION
**Pull request by dkiatsongsak00**
- apply colors
- apply font size
- style bar chart
- add special badge conditional see more
- hide header and add back button

### CODE REVIEW

File: src/components/containers/BarChart.js [hunk 1]
```diff
@@ -4,32 +4,60 @@ import {
...
+  PRIMARY_MEDIUM,
+  PRIMARY_DARK,
+  TXT_MEDIUM_BG,
+} from "../../constants/colorCodes";
 
 const BarChart = ({ data, averageValue }) => {
-  const ValueLabel = (props) => {
-    const { datum } = props;
-    return <VictoryLabel {...props} text={datum.value} />;
+  const ValueLabel = () => null;
```
Comment (rena52107) [inline src/components/containers/BarChart.js:17]: Is this function necessary?

File: src/components/containers/BarChart.js [hunk 2]
```diff
@@ -4,32 +4,60 @@ import {
...
+            tickLabels: { fill: TXT_MEDIUM_BG, fontSize: 12 },
+            grid: { stroke: "none" },
+          }}
+          tickFormat={formatYAxisLabel}
+        />
         <VictoryBar
           data={data}
           x="label"
           y="value"
           labelComponent={<ValueLabel />}
```
Comment (rena52107) [inline src/components/containers/BarChart.js:48]: If the function above is not necessary, I guess you can remove this prop.

**Review by rena52107 — COMMENTED** [review]
Left some comment, please request again after reviewing the comments

File: src/components/containers/BarChart.js (hunk 1 above)
Comment (dkiatsongsak00) [reply src/components/containers/BarChart.js:17]: Yes, it is necessary to remove the value displayed on top of the bar

**Review by rena52107 — APPROVED** [review]
Review completed

**Pull request merged**